UKHomeOffice/cerberus-service
# COP-10872 - View pnr data post login request

## Description
This PR adds the requirement to answer if a user wants to view PNR data during their keycloak session. This happens at the start of their session immediately after login.

## To Test
- Pull & run against dev
- If logged in, sign out.
- if already signed out, login, immediately after login, a form will be presented (see image below)

![image](https://user-images.githubusercontent.com/19333750/170712062-1c2dbab7-66c7-4aa5-ba51-7dcff751ec8b.png)


## Developer Checklist

\* Required
